test/strategies/cloudsearch.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+// Load external modules
+const Lab = require('lab');
+const MockAWS = require('mock-aws');
+
+// Load internal modules
+const CloudSearchStrategy = require('../../src/strategies/cloudsearch');
+
+// Test shortcuts
+const lab = exports.lab = Lab.script();
+const expect = Lab.assertions.expect;
+
+let strategy;
+
+lab.after((done) => {
+  MockAWS.restore();
+  done();
+});
+
+lab.before((done) => {
+  strategy = new CloudSearchStrategy({
+    region: 'us-west-1',
+    endpoint: 'https://www.ideapod.com'
+  });
+  done();
+});
+
+lab.describe('CloudSearchStrategy', () => {
+  lab.describe('uploadDocuments', () => {
+    lab.beforeEach((done) => {
+      MockAWS.mock('CloudSearchDomain', 'uploadDocuments', (params, call) => {
```

Comment:
If `call` refers to `callback`, use `callback` as the name`, `call` is misleading.